02-household-queries/requirements.in

```diff
@@ -16,3 +16,7 @@ langchain-text-splitters
 pdfminer.six
 rapidocr-onnxruntime
 sentence-transformers
+nltk
+setuptools
+wheel
+spacy
```

Comment:
When I try to re-run `pip-compile requirements.in` (so that spacy is added to `requirements.txt`), it seems to hang.
Also why is `setuptools` and `wheel` needed?

Reply:
I had some issues with the spacy installation that seemed to have a dependency on wheel and setup tools, taking a another look